### PR TITLE
DSDEGP-2219 Persist AWS auth tokens during a session

### PR DESCRIPTION
**Pull request by vvicario**
Persist AWS auth tokens during a session

----

### Checklist

- [ X ] **Submitter**: Include the JIRA issue number in the PR description
- [ X ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ X ] **Submitter**: Ensure that you have added or updated tests
- [ X ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=189431877).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: src/app/file-manager/file-export-modal/file-export-modal.component.ts [hunk 1]
```diff
@@ -91,17 +101,17 @@ export class FileExportModalComponent implements OnInit {
     }
   }
 
-  formChange(): boolean {
+  disableExportButton(): boolean {
```
Comment (pshapiro4broad) [inline src/app/file-manager/file-export-modal/file-export-modal.component.ts:102]: With this change do you still need `disableExport` as a field? It looks like you compute it completely in this method. If so, you can make `disableExport` a local var and remove the field and references to it in this class.

Comment (vvicario) [reply src/app/file-manager/file-export-modal/file-export-modal.component.ts:102]: You are right, we don't need disable export as a field. I just removed it

**Review by pshapiro4broad — APPROVED** [review]
The change looks OK, but the travis build is failing, not sure why. The build needs to be fixed before the branch can be merged.

**Comment by vvicario** [comment]
Sorry @pshapiro4broad I merged before I read your last comment

**Comment by vvicario** [comment]
@pshapiro4broad It works fine on develop,
https://travis-ci.org/broadinstitute/firecloud-explorer/builds/363118747

**Pull request merged**